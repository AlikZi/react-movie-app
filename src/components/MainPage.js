import React from 'react';
import '../styles/styles.scss';

const MainPage = () => (
  <div>
    <h1>Movie List</h1>
  </div>
);

export default MainPage;

/*constructor(props) {
    super(props);
    this.componentDidMount = this.componentDidMount.bind(this);
    this.state = {movies: []};
  }

  componentDidMount(){
    fetch(`http://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&page=1`)
      .then( res => res.json())
      .then( data => this.setState(() => ({movies: data['results']})))
      .catch(err => 
        console.log(err));
    }

    const moviesItems = this.state.movies.map(movie => (
      <div key={movie.id}>
        <p>{movie.title}</p>
        <p>{movie.overview}</p>
        <img class="card-image" alt={movie.title} src={URL_IMAGE + movie.poster_path}/>
      </div>
    ))
*/