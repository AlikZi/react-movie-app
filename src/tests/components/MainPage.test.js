import React from 'react';
import { shallow } from 'enzyme';
import MainPage from "../../components/MainPage";

test('should render Header correctly', () => {
    const wrapper = shallow(<MainPage />);
    expect(wrapper).toMatchSnapshot();
}); 