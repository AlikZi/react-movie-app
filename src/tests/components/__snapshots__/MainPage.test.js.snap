// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="container"
>
  <div
    className="main"
  >
    <Connect(MoviesList) />
  </div>
</div>
`;
